qa/suites/fs/upgrade/upgrade_without_reducing_max_mds/tasks/3-upgrade-with-workload.yaml

```diff
@@ -0,0 +1,71 @@
+tasks:
+- parallel:
+  - upgrade-tasks
+  - workload-tasks
+
+upgrade-tasks:
+  sequential:
+  - cephadm.shell:
+      env: [sha1]
+      host.a:
+        - ceph config set mon mon_warn_on_insecure_global_id_reclaim false --force
+        - ceph config set mon mon_warn_on_insecure_global_id_reclaim_allowed false --force
+        - ceph config set global log_to_journald false --force
+        - ceph orch ps
+        - ceph versions
+        - ceph -s
+        - ceph orch ls
+        - ceph orch daemon redeploy "mgr.$(ceph mgr dump -f json | jq .standbys | jq .[] | jq -r .name)" --image quay.ceph.io/ceph-ci/ceph:$sha1
```

Comment:
I presume you're doing this to force upgrading the mgr to something newer than 16.2.4 so you can do staggered upgrades? In this case, don't test `fail_fs` unless the `ceph-mgr` is at least v16.2.10. Suggest structuring it this way (options are limited with yaml matrices):

```
diff --git a/qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/pacific/% b/qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/pacific/%
new file mode 100644
index 00000000000..e69de29bb2d
diff --git a/qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/pacific/fail_fs/no.yaml b/qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/pacific/fail_fs/no.yaml
new file mode 100644
index 00000000000..e69de29bb2d
diff --git a/qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/pacific/fail_fs/yes.yaml b/qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/pacific/fail_fs/yes.yaml
new file mode 100644
index 00000000000..53b976b873f
--- /dev/null
+++ b/qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/pacific/fail_fs/yes.yaml
@@ -0,0 +1,5 @@
+overrides:
+  ceph:
+    conf:
+      mgr:
+        mgr/orchestrator/fail_fs: true
diff --git a/qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/pacific.yaml b/qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/pacific/head.yaml
similarity index 100%
rename from qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/pacific.yaml
rename to qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/pacific/head.yaml
```

Reply:
I suspect your upgrade problems stem from doing this forced redeploy to a newer mgr version. Try again with the above changes.

Reply:
As discussed in IRC, this doesn't work as intended. Therefore closing this conversation.